R/loadFromMetaphlan.R

```diff
@@ -149,20 +149,21 @@ loadFromMetaphlan <- function(file, sample_meta = NULL, phy_tree = NULL, ...){
 }
 
 # Check that metaphlan file contains correct information
-.check_metaphlan <- function(data){
-    # Get rowdata columns
-    rowdata_columns <- data[ , 1:2]
-    # Get columns that go to assay
-    assay_columns <- data[ , 3:ncol(data)]
+.check_metaphlan <- function(x){
+    # retrieving field types present in the data 
+    col_types <- sapply(x, class)
+    if(any(!(col_types %in% c("character", "numeric")))) {
+      warning("File contains other than character and numeric fileds.")
```

Comment:
The file does not contain those but the R object that was read from the file.

How about this:
"The input data should not contain other than character and numeric fields."